### PR TITLE
fix: Cloudflare worker environment variable persistence and document CDN endpoint(s) (DIA-848)

### PR DESCRIPTION
**Pull request by joeyAghion**
This adds basic documentation of the alternate, edge-cached Metaphysics endpoints.

Important bonus: fix the `wrangler` deploy command to avoid wiping out configured environment variables upon deployments (as discussed in https://github.com/cloudflare/workers-sdk/issues/276).

### CODE REVIEW

File: docs/cdn.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+Metaphysics can be queried through its CDN endpoint, `metaphysics-[staging|production]-alt.artsy.net`.
```
Comment (damassi) [inline docs/cdn.md:1]: Not for this PR obviously, but should we talk about promoting the name of this endpoint to something more self-describing? `alt` feels a bit ambiguous.

Comment (joeyAghion) [reply docs/cdn.md:1]: Yeah I had a similar thought... `-cdn`? And maybe it's time to finally drop `-production` from this app's hostname.

Comment (damassi) [reply docs/cdn.md:1]: (We should think about tackling this before this work starts to spread to Eigen)

**Comment by mzikherman** [comment]
Thanks for this!

**Pull request merged**